### ISSUE TITLE
Lack of manual testing procedure for accessibility

### ISSUE THREAD

**Issue by Heydon** [iTwin/stratakit#817]
We currently do not have a procedure in place for manually testing components for accessibility (specifically WCAG compliance). Manual testing catches significantly more accessibility issues than automated testing.

A clear and robust procedure will mean component accessibility can be tested _by_ anyone and tracked directly within Github and our issues stream.

I'm proposing we make use of a Github [issue form](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms) to both 

* provide instructions on undertaking the testing,
* output a structured report on the findings, per component.

Issue forms support checkbox input, so each individual test can be recorded as a Boolean pass/fail. Notes can be provided via textarea to add additional context and suggestions for remediation (fixes).

Issues created using this template/form can then be tracked as normal or organized into a dedicated Github project.
